docs/dptp-triage-sop/misc.md

```diff
@@ -0,0 +1,23 @@
+## Probe Failing on ci-rpms
+
+```
+[FIRING:1] ProbeFailing blackbox (https://artifacts-rpms-openshift-origin-ci-rpms.apps.ci.l2s4.p1.openshiftapps.com/openshift-origin-v3.11/repodata/repomd.xml prow-monitoring/prow critical)
+Probing the instance https://artifacts-rpms-openshift-origin-ci-rpms.apps.ci.l2s4.p1.openshiftapps.com/openshift-origin-v3.11/repodata/repomd.xml has been failing for the past minute.
+```
+
+The TP team does not own these services.
```

Comment:
We have a card to fix this long-term too: https://issues.redhat.com/browse/DPTP-2981